examples/compute/instance/default_secgroup/main.tf

```diff
@@ -0,0 +1,18 @@
+resource "vkcs_compute_instance" "front_worker" {
+  name      = "front-worker-${count.index}-tf-example"
```

Comment:
no count - no name template

Reply:
Should also remove dynamic block from lb members?

Reply:
done